src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -2628,6 +2630,36 @@ def managed_db_restore(
         parameters=kwargs)
 
 
+def managed_db_restore_ltr_backup(
+        cmd,
+        client,
+        database_name,
+        managed_instance_name,
+        resource_group_name,
+        backup_id,
```

Comment:
why do you need database_name, managed_instance_name and resource_group_name?

It looks like backup_id is the ltr backup resource id ?

Reply:
yes, backup_id is the ltr backup resource id 

the other *_name variables should all represent target resources.  fix on the way.